how-to-guides/consensus-node.md

```diff
@@ -342,12 +342,69 @@ Optional: If you would like celestia-app to run as a background process, you can
 
 ### Optional: Reset network
 
-This will delete all data folders so we can start fresh:
+There are several ways to reset your consensus node, depending on what you need to preserve:
+
+#### Option 1: Reset blockchain data with `reset-state`
+
+This command removes blockchain data but preserves validator state and address book:
+
+```sh
+celestia-appd tendermint reset-state
+```
+
+This preserves your configuration, validator state (`priv_validator_state.json`), and peer connections (`addrbook.json`), but removes:
+- blockstore.db
+- state.db
+- evidence.db
+
+This option is safer for validator nodes and was created specifically for validators who need to reset without losing their validator state.
+
+#### Option 2: Full reset with `unsafe-reset-all`
+
+⚠️ **CAUTION FOR VALIDATORS**: This is considered "unsafe" because it resets validator state:
 
 ```sh
 celestia-appd tendermint unsafe-reset-all --home $HOME/.celestia-app
 ```
 
+This command:
+- Resets blockchain data
+- Resets validator state (`priv_validator_state.json`) 
+- Clears address book (`addrbook.json`)
+- Preserves node configuration
+
+Only use this during upgrades or when the chain is not moving. Always backup your validator keys before using this command.
```

Comment:
Is this really supposed to be used during upgrades? It's not clear at all when this is actually supposed to be used. What does "chain is not moving" mean? Should they go to Discord first to check if that's the case?

Reply:
AI garble. fixing up...